coordinator-service/src/disk-coordinator.ts

```diff
@@ -78,6 +84,36 @@ export class DiskCoordinator implements Coordinator {
         return this.db
     }
 
+    getParameters(): CeremonyParameters {
+        return clonedeep(this.db.parameters)
+    }
+
+    getContributorChunks(contributorId: string): ChunkInfo[] {
+        const ceremony = this.db
+        return ceremony.chunks.map(({ lockHolder, chunkId, contributions }) => {
+            return {
+                lockHolder,
+                chunkId,
+                contributed: contributions.some(
+                    (a) => a.contributorId == contributorId,
+                ),
+            }
+        })
+    }
+
+    getVerifierChunks(contributorId: string): ChunkInfo[] {
+        const ceremony = this.db
+        return ceremony.chunks.map(({ lockHolder, chunkId, contributions }) => {
+            return {
+                lockHolder,
+                chunkId,
+                contributed: contributions.some(
+                    (a) => a.verifierId == contributorId,
```

Comment:
I believe the verifier is looking for a different view - it looks for all the chunks where the last contribution is not verified (or any contribution in that chunk, for that matter. But the last contribution should be the only one, it's supposed to be an invariant of the system). That way it knows it should lock the chunk and work on it. Any verifier can work on that chunk.